src/pudl/transform/ferc1.py

```diff
@@ -853,6 +1024,85 @@ def strip_col_of_non_ints(self, col: pd.Series) -> pd.DataFrame:
         )
 
 
+class PlantInServiceFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """A transformer for the Electric Plant in Service table."""
+
+    table_id: Ferc1TableId = Ferc1TableId.PLANT_IN_SERVICE_FERC1
+
+    @cache_df(key="dbf")
+    def align_row_numbers_dbf(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Align historical FERC1 DBF row numbers with XBRL account IDs."""
+        return pd.merge(
+            df,
+            get_ferc1_dbf_xbrl_glue(self.source_table_id(Ferc1Source.DBF)),
+            on=["report_year", "row_number"],
+        ).rename(columns={"xbrl_column_stem": "ferc_account_id"})
```

Comment:
It's the sometimes very ong DB FERC XBRL column name, but without the suffix which indicates which type of value the original XBRL fact was reporting. I don't know what all columns we're going to end up with in here, and the meaning of the `xbrl_column_stem` is going to change from table to table, so having some kind of rename in here somewhere that makes it meaningful in the context of the table seemed like it made sense.  I went with calling it the `id` since it will be the primary key in the table that contains other metadata like the account numbers.

Reply:
when i see `id` (especially in the context of the ferc account id) i assume this is a number, not a label that maybe has the actually number in it. I suggest calling this `ferc_account_description` or `ferc_account_label`

Reply:
I think `ferc_account_label` is probably right, given the way we use label in the encoding tables etc. The column that has the number can stay "ferc_account". I'll update this.

Reply:
I've made this change.